Sources/MapboxCoreNavigation/PredictiveCacheLocationOptions.swift

```diff
@@ -0,0 +1,38 @@
+
+import Foundation
+import MapboxNavigationNative
+
+/// `PredictiveCacheLocationOptions` controls various configurations for a `Predictive Caching` mechanic.
+public class PredictiveCacheLocationOptions {
```

Comment:
This class seems to be about more than just the location of the predictive cache. The “location” in PredictiveLocationTrackerOptions refers to predictive location tracking, so PredictiveLocationOptions or PredictiveCacheOptions would probably be more accurate.

Reply:
Agree, renamed to `PredictiveCacheOptions`.